micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/metrics/BpmnExecutionMetrics.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package info.novatec.micronaut.camunda.bpm.feature.metrics;
+
+import io.micronaut.context.ApplicationContext;
+import org.camunda.bpm.engine.management.Metrics;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class BpmnExecutionMetrics extends CamundaMetrics {
+
+    private long rootProcessInstanceStart = 0;
```

Comment:
If the application redeploys, you may end up with the value being zero again, which is not true.